### PR TITLE
Storybook: Refactor ComposedChart.stories.tsx

### PR DESCRIPTION
**Pull request by nikolasrieble**
## Description
- Use a single shared render function.
- Collect all stories for the component in a single file. 
- Add a README to refer to. 

- Delete the time scale example of a ComposedChart
The motivation to delete the time scale example is that the time scale behaviour is a function the XAxis. This behaviour should be shown in great detail on the XAxis story.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by nikolasrieble** [comment]
@proke03 Please consider my refactoring. I followed the pattern described in https://storybook.js.org/blog/storybook-csf3-is-here to defined a template rendering function.

If we define such a template, then every story differs only in details, which helps the user better understand what is happening.

File: storybook/stories/chart/ComposedChart.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+export default {
+  title: 'Recharts/Chart/Composed Chart',
+  component: Composed,
+};
+
+// This render template can be shared across multiple stories.
+// All stories use the same data, but pass different children.
+const HorizontalChartTemplate = {
+  render: (args: any) => {
```
Comment (ckifer) [inline storybook/stories/chart/ComposedChart.stories.tsx:29]: I hate this `any` here but know why its needed 😅 

Could we maybe maybe this `Record<string, any>`? Just for a little bit of a stricter type

**Review by ckifer — APPROVED** [review]
Looks fine, my comment is a nit 🚀

**Comment by proke03** [comment]
Thanks for making a good standard.

File: storybook/stories/chart/ComposedChart.stories.tsx (hunk 1 above)
Comment (nikolasrieble) [reply storybook/stories/chart/ComposedChart.stories.tsx:29]: It was late, and I had to submit for review :D I agree, and will iterate.

**Pull request merged**